engine/src/main/java/org/terasology/world/block/tiles/WorldAtlasImpl.java

```diff
@@ -275,6 +303,23 @@ private void createTextureAtlas(final Texture texture) {
         return data;
     }
 
+    // Ref: http://stackoverflow.com/questions/221830/set-bufferedimage-alpha-mask-in-java/8058442#8058442
+    public void storeGreyscaleMapIntoAlpha(BufferedImage imageWithoutAlpha, BufferedImage greyscaleImage) {
+        int width = imageWithoutAlpha.getWidth();
+        int height = imageWithoutAlpha.getHeight();
+
+        int[] imagePixels = imageWithoutAlpha.getRGB(0, 0, width, height, null, 0, width);
```

Comment:
There were a few different snippets of how to do this online.  The array method here seemed like the most efficient without getting too complicated.